container_fixtures/postgres/init_wanda_db.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" <<-EOSQL
+  CREATE DATABASE wanda_db;
+  CREATE ROLE wanda_user WITH LOGIN PASSWORD 'wanda_pass';
```

Comment:
do we need other extra user?
this is dev, I think we could reuse the `postgres` user. 
It is easier to debug with tools like `dbeaver`